components/odh-notebook-controller/controllers/notebook_controller_test.go

```diff
@@ -646,6 +646,15 @@ var _ = Describe("The Openshift Notebook controller", func() {
 									},
 								},
 							},
+							{
+								Name: "oauth-client",
+								VolumeSource: corev1.VolumeSource{
+									Secret: &corev1.SecretVolumeSource{
+										SecretName:  Name + "-oauth-client-generated",
+										DefaultMode: pointer.Int32Ptr(420),
```

Comment:
@daniellutz the idea here is that the linter only runs on new code, so that's why it is not flagging the existing usages of `pointer.Int32Ptr`. What you should do here is to simply use `ptr.To` instead of this, and it will work just fine. The idea is that ptr.To is a better replacement for the old functions, and it could not be used before because it requires go 1.18+ features https://pkg.go.dev/k8s.io/utils/ptr#section-readme